src/index.html

```diff
@@ -1,12 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <main class="wrapper">
+      <div class="calendar calendar--start-day-mon calendar_month_lenght--29">
+        <div class="calendar_day"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
Fix everywhere, we're using `__` for BEM block elements